ledger/src/blockstore_processor.rs

```diff
@@ -480,6 +483,9 @@ fn do_process_blockstore_from_root(
         },
     );
     assert!(bank_forks.active_banks().is_empty());
+    if !bank_forks.root_bank().verify_bank() {
```

Comment:
Without this check, validators/cluster might continue to create newer blocks combined with automatic restart practiced from the ops perspectives. That's because every restarts delays next snapshot slot by the snapshot interval (usually 100). ;)

Reply:
Well, I'll do write this in-souce comment as well.